wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -132,6 +137,23 @@ class SwiftGenerator private constructor(
       else -> null
     }
 
+  // see https://protobuf.dev/programming-guides/proto3/#default
+  private val Field.proto3InitialValue: String
+    get() = when {
+      isMap -> "[:]"
+      isRepeated -> "[]"
+      isOptional -> "nil"
+      isEnum -> "." + enumType.constants.first().name
```

Comment:
This is kind of dangerous. The correct value is `EnumType(rawValue: 0)`.

You can always prune an enum value which also makes things a little funky

Reply:
Is it valid to prune the default value? If no, is there currently any protection against it?

I chose this route because it makes the generated code _much_ nicer to read, but I can see why `rawValue` is probably where this should end up.

Reply:
In all cases we want to conform to the [proto spec](https://protobuf.dev/programming-guides/proto3/#enum), so `EnumType(rawValue: 0)!` is right. If you do prune it then this force unwrap would fail.

This could also be the right place to guard against pruning it...if we can't find the zero-value here, then throw an exception and have the compiler bail out. I believe the prunes have already happened at this point (but we should verify).

Reply:
Personally I have been pruning my Proto2 enums' 0 value since they're all optional, but yeah if it's required then you're kinda stuck with rawValue: 0

Reply:
I removed this and now use `rawValue:0` inside of the new `defaultIfMissing`